src/test/java/org/opendatadiscovery/oddrn/model/ODDPlatformDataEntityGroupPathTest.java

```diff
@@ -0,0 +1,31 @@
+package org.opendatadiscovery.oddrn.model;
+
+import org.junit.jupiter.api.Test;
+import org.opendatadiscovery.oddrn.AbstractGeneratorTest;
+import org.opendatadiscovery.oddrn.exception.EmptyPathValueException;
+
+public class ODDPlatformDataEntityGroupPathTest extends AbstractGeneratorTest {
+
+    @Test
+    public void shouldGenerateDataEntityGroupPath() throws Exception {
+        shouldGeneratePath(
+                ODDPlatformDataEntityGroupPath.builder()
+                        .type("KAFKA_SERVICE")
+                        .id(1L)
+                        .build(),
+                "id",
+                "//oddplatform_deg/type/KAFKA_SERVICE/dataEntityGroupId/1"
```

Comment:
why would we need to have a type in the oddrn?
why won't we just do '//oddplatform_deg/id/1'?